scalability_and_performance/cnv-performance-scale-resilience-practices/recommended-node-practices.adoc

```diff
@@ -0,0 +1,86 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="recommended-node-practices"]
+= Recommended node practices
+include::_attributes/common-attributes.adoc[]
+:context: recommended-node-practices
+
+toc::[]
+
+This topic provides recommended node practices for a Container Native Virtualization environment to be performant, scalable and resilient in {product-title}.
+
+If a node fails and Node Health Checks operator is not deployed  and configured with an appropriate remediation operator - Self Node Remediation (SNR) and/or Fence Agents Remediation (FAR) on your cluster, Virtual Machines (VMs) will encounter downtime until the node recovers as they are not automatically recovered to other available nodes by default.
+
+Here are the configurations recommended to avoid virtual machines (VMs) downtime on a critical environment and for your workloads to be resilient under node failure conditions:
```

Comment:
```suggestion
Here are the configurations recommended to avoid VMs downtime on a critical environment and for your workloads to be resilient under node failure conditions:
```
You have already defined the term above. You can use it the rest of the document.